### PR TITLE
Update Vkontakte.php

### PR DESCRIPTION
**Pull request by WinterSilence**
Fix api request in `getUserProfile`: adds birthday, country, city, 
Add constant `API_VERSION`: usefull in api requests.

| Q                        | A
| ------------------------ | ---
| Fixed Issues?            | no
| Patch: Bug Fix?          | yes
| Major: Breaking Change?  | no
| Minor: New Feature?      | yes

### CODE REVIEW

**Comment by StorytellerCZ** [comment]
Would be nice if scope beyond the needed minimum could be defined in options, other than that LGTM.

**Comment by WinterSilence** [comment]
@StorytellerCZ what is LGTM?

File: src/Provider/Vkontakte.php [hunk 1]
```diff
@@ -21,53 +21,51 @@
...
 
     /**
      * {@inheritdoc}
      */
-    protected $accessTokenUrl = 'https://api.vk.com/oauth/token';
+    protected $accessTokenUrl = 'https://oauth.vk.com/access_token';
 
     /**
      * {@inheritdoc}
+     * @todo Overwrite methods setUserStatus/getUserActivity(status), getUserPages/setPageStatus(pages),
```
Comment (ApacheEx) [inline src/Provider/Vkontakte.php]: We do not need this here in PR.

File: src/Provider/Vkontakte.php [hunk 2]
```diff
@@ -104,33 +104,32 @@ public function getUserProfile()
 
         $data = new Collection($response->response[0]);
 
-        if (!$data->exists('id')) {
+        if (! $data->exists('id')) {
             throw new UnexpectedApiResponseException('Provider API returned an unexpected response.');
         }
 
-        $userProfile = new Profile();
+        $userProfile = new User\Profile;
```
Comment (ApacheEx) [inline src/Provider/Vkontakte.php]: any reason to change it?

File: src/Provider/Vkontakte.php [hunk 3]
```diff
@@ -1,16 +1,16 @@
 <?php
 /*!
-* Hybridauth
-* https://hybridauth.github.io | https://github.com/hybridauth/hybridauth
-*  (c) 2017 Hybridauth authors | https://hybridauth.github.io/license.html
-*/
+ * Hybridauth
+ * https://hybridauth.github.io | https://github.com/hybridauth/hybridauth
+ *  (c) 2017-2019 Hybridauth authors | https://hybridauth.github.io/license.html
```
Comment (ApacheEx) [inline src/Provider/Vkontakte.php]: do not change it

File: src/Provider/Vkontakte.php [hunk 4]
```diff
@@ -21,53 +21,51 @@
  *       'callback'  => Hybridauth\HttpClient\Util::getCurrentUrl(),
  *       'keys'      => ['id' => '', 'secret' => ''],
  *   ];
- *
```
Comment (ApacheEx) [inline src/Provider/Vkontakte.php]: I don't think we need to change something here in this description.

File: src/Provider/Vkontakte.php (hunk 2 above)
Comment (WinterSilence) [reply src/Provider/Vkontakte.php]: @ApacheEx more readable, other providers use too call as `User\Profile`

File: src/Provider/Vkontakte.php (hunk 1 above)
Comment (WinterSilence) [reply src/Provider/Vkontakte.php]: @ApacheEx why it's normal practice

File: src/Provider/Vkontakte.php (hunk 4 above)
Comment (WinterSilence) [reply src/Provider/Vkontakte.php]: @ApacheEx why?

File: src/Provider/Vkontakte.php (hunk 1 above)
Comment (ApacheEx) [reply src/Provider/Vkontakte.php]: normal practice where? for all "future" changes please create separate PR and describe goal of your changes.

File: src/Provider/Vkontakte.php (hunk 2 above)
Comment (ApacheEx) [reply src/Provider/Vkontakte.php]: then, let's use it like in other providers.  `$userProfile = new User\Profile();`

File: src/Provider/Vkontakte.php [hunk 5]
```diff
@@ -104,33 +104,32 @@ public function getUserProfile()
 
         $data = new Collection($response->response[0]);
 
-        if (!$data->exists('id')) {
+        if (! $data->exists('id')) {
```
Comment (ApacheEx) [inline src/Provider/Vkontakte.php]: please keep it as it was and do not use extra space for future `if` statements.

File: src/Provider/Vkontakte.php (hunk 1 above)
Comment (WinterSilence) [reply src/Provider/Vkontakte.php]: @ApacheEx [everywhere](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#515-todo)

File: src/Provider/Vkontakte.php (hunk 2 above)
Comment (WinterSilence) [reply src/Provider/Vkontakte.php]: @ApacheEx pls stop this shit, it's not important

**Comment by ApacheEx** [comment]
won't be merged as lots of unnecessary changes made.

File: src/Provider/Vkontakte.php (hunk 5 above)
Comment (WinterSilence) [reply src/Provider/Vkontakte.php]: @ApacheEx pls, publicate your code style conversation

**Pull request closed without merging**